sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/tests/QuestionAnsweringClientLiveTests.cs

```diff
@@ -65,51 +65,51 @@ public async Task AnswersFollowupKnowledgebaseQuestion()
                 IncludeUnstructuredSources = true,
             };
 
-            Response<KnowledgebaseAnswers> response = await Client.QueryKnowledgebaseAsync(TestEnvironment.ProjectName, options, TestEnvironment.DeploymentName);
+            Response<KnowledgeBaseAnswers> response = await Client.QueryKnowledgeBaseAsync(TestEnvironment.ProjectName, options, TestEnvironment.DeploymentName);
 
             Assert.That(response.GetRawResponse().Status, Is.EqualTo(200));
             Assert.That(response.Value.Answers.Count, Is.EqualTo(2));
 
-            IList<KnowledgebaseAnswer> answers = response.Value.Answers.Where(answer => answer.ConfidenceScore > 0.6).ToList();
+            IList<KnowledgeBaseAnswer> answers = response.Value.Answers.Where(answer => answer.ConfidenceScore > 0.6).ToList();
             Assert.That(answers, Has.Count.EqualTo(1));
-            Assert.That(answers, Has.All.Matches<KnowledgebaseAnswer>(answer => answer.Id == 23 && answer.AnswerSpan.Text == "two to four hours"));
+            Assert.That(answers, Has.All.Matches<KnowledgeBaseAnswer>(answer => answer.Id == 23 && answer.AnswerSpan.Text == "two to four hours"));
         }
 
         [RecordedTest]
-        public void QueryKnowledgebaseBadArgument()
+        public void QueryKnowledgeBaseBadArgument()
         {
-            KnowledgebaseQueryOptions options = new(" ");
+            QueryKnowledgeBaseOptions options = new(" ");
 
-            RequestFailedException ex = Assert.ThrowsAsync<RequestFailedException>(async () =>
+            RequestFailedException ex = Assert.ThrowsAsync<RequestFailedException>((AsyncTestDelegate)(async () =>
```

Comment:
ohh I have never seen `AsyncTestDelegate`. Would you mind explaining to me what it is?

Reply:
That's the parameter type to `Assert.ThrowsAsync`, though I don't remember typing this - no need to since the compiler can infer it. Wonder if some refactoring thing I did for this PR did it automatically.

Reply:
...that said, I'll remove it. Not necessary and just adds confusion.